hnn_core/tests/test_compare_hnn.py

```diff
@@ -21,12 +23,23 @@ def test_hnn_core():
 
     hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
 
+    # default params
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
     params = read_params(params_fname)
 
-    net = Network(params, n_jobs=1)
-    dpl = simulate_dipole(net)[0]
+    # run the simulation
+    net = Network(params)
+
+    if backend == 'mpi':
+        with MPIBackend(n_jobs=n_jobs, n_procs=2, mpi_cmd='mpiexec'):
+            dpl = simulate_dipole(net)[0]
```

Comment:
just a thought, since you are doing only a smoke test -- we could simulate on a smaller network so that the test is fast. 

I would also add a test to check that the output of both the backends is the same. Again, smaller network should be fine.

Reply:
The intention of this is to actually check the dipole and spiking data with MPI to make sure that it matches the sequential version. Each of the three tests has value in confirming correct results. This will need some adjustment when testing multiple trials, but getting consistent results with MPI vs. joblibs will be important. Perhaps in that PR `test_mpi` and `test_joblib` will run 3 trials and `test_hnn_core` can still run one trial.

Reply:
I'm a bit afraid that we'll start hitting the CI time limits if we don't optimize tests from the beginning. If we check the outputs against each other on a smaller network, it's legit, no?

Again could be another PR since we are already too big in this PR. Let's just keep track of the issues somewhere

Reply:
The limit for Travis is 40 minutes, but starting small makes sense. Can we delay making reduced tests until we can verify equal results with HNN once this PR and another for seeding of trials have been implemented?

Reply:
sure! let's just open issues. It will also be helpful to have some of these issues for new contributors in a coding sprints. For onboarding them into the development cycle.